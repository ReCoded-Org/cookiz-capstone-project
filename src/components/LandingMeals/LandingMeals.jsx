const LandingMeals = () => {
  return (
    <div className="grid md:grid-cols-6 m-14 ">
      <div className="md:col-span-1 " >
        <nav>
          <div >
            <div className="flex justify-center mt-[-14px]">
              <input className="rounded-xl  font-semibold hover:font-bold" type="search" placeholder="Search" name="search" />
             </div>
            <h1 className=  "flex justify-center font-bold pt-8">Categories</h1>
            <ul className="flex items-center flex-col" >

           
              <li className="font-sans">item 1</li>
              <li>item 2</li>
              <li>item 3</li>
              <li>item 4</li>
              <li>item 5</li>
            </ul>
          </div>

        </nav>
        </div>
        <div className="md:col-span-5  ">
        <div className="mt-8 grid md:grid-cols-3 gap-10 m-16" >

          <div className="card">
            <img className="w-full sm:h-48 object-cover " src="images/meat.jpg" alt="meatball" />

            <span className="m-4 font-bold text-sm md:text-base ">Spanish Paella </span>


            <div>
              <img className="inline p-2 md:w-11  w-8" src="images/Avatar.png" alt="chef" />
              <span className="text-grey-500 md:text-sm font-normal text-xs" >Jennifer Lawrence</span>
            </div>
            <div className=" badge"> <span >60 tl</span></div>


          </div>
          <div className="card">
            <img className="w-full sm:h-48 object-cover " src="images/meat.jpg" alt="meatball" />

            <span className="m-4 font-bold text-sm md:text-base">Spanish Paella </span>


            <div>
              <img className="inline p-2 md:w-11  w-8 " src="images/Avatar.png" alt="chef" />
              <span className="text-grey-500 md:text-sm text-xs" >Jennifer Lawrence</span>
            </div>
            <div className="badge"> <span >60 tl</span></div>


          </div>
          <div className="card">
            <img className="w-full sm:h-48 object-cover " src="images/meat.jpg" alt="meatball" />

            <span className="m-4 font-bold text-sm md:text-base">Spanish Paella </span>


            <div>
              <img className="inline p-2 md:w-11  w-8" src="images/Avatar.png" alt="chef" />
              <span className="text-grey-500 md:text-sm text-xs" >Jennifer Lawrence</span>
            </div>
            <div className="badge"> <span >60 tl</span></div>


          </div>
          <div className="card">
            <img className="w-full sm:h-48 object-cover " src="images/meat.jpg" alt="meatball" />

            <span className="m-4 font-bold text-sm md:text-base">Spanish Paella </span>


            <div>
              <img className="inline p-2 md:w-11  w-8 " src="images/Avatar.png" alt="chef" />
              <span className="text-grey-500 md:text-sm text-xs" >Jennifer Lawrence</span>
            </div>
            <div className="badge"> <span >60 tl</span></div>


          </div>
          <div className="card">
            <img className="w-full h-32 sm:h-48 object-cover " src="images/meat.jpg" alt="meatball" />

            <span className="m-4 font-bold text-sm md:text-base">Spanish Paella </span>


            <div>
              <img className="inline p-2 md:w-11  w-8 " src="images/Avatar.png" alt="chef" />
              <span className="text-grey-500 md:text-sm text-xs" >Jennifer Lawrence</span>
            </div>
            <div className="badge"> <span >60 tl</span></div>


          </div>
          <div className="card">
            <img className="w-full h-32 sm:h-48 object-cover" src="images/meat.jpg" alt="meatball" />

            <span className="m-4 font-bold text-sm md:text-base">Spanish Paella </span>


            <div>
              <img className="inline p-2 md:w-11  w-8  " src="images/Avatar.png" alt="chef" />
              <span className="text-grey-500 md:text-sm text-xs" >Jennifer Lawrence</span>
            </div>
            <div className="badge"> <span >60 tl</span></div>


          </div>

        </div>
        </div>
      
    </div>
  );
}

export default LandingMeals;