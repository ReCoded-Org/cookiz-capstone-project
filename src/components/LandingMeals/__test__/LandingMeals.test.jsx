
import renderer from "react-test-renderer";

import LandingMeals from "../LandingMeals";


it("renders correctly", () => {
  const tree = renderer.create(<LandingMeals />).toJSON();
  expect(tree).toMatchSnapshot();
});