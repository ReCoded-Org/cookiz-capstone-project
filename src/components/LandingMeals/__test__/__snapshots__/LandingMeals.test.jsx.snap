// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="grid md:grid-cols-6 m-14 "
>
  <div
    className="md:col-span-1 "
  >
    <nav>
      <div>
        <div
          className="flex justify-center mt-[-14px]"
        >
          <input
            className="rounded-xl font-semibold hover:font-bold"
            name="search"
            placeholder="Search"
            type="search"
          />
        </div>
        <h1
          className="flex justify-center font-bold pt-8"
        >
          Categories
        </h1>
        <ul
          className="flex items-center flex-col"
        >
          <li
            className="font-sans"
          >
            item 1
          </li>
          <li>
            item 2
          </li>
          <li>
            item 3
          </li>
          <li>
            item 4
          </li>
          <li>
            item 5
          </li>
        </ul>
      </div>
    </nav>
  </div>
  <div
    className="md:col-span-5 "
  >
    <div
      className="mt-8 grid md:grid-cols-3 gap-10 m-16"
    >
      <div
        className="card"
      >
        <img
          alt="meatball"
          className="w-full sm:h-48 object-cover "
          src="images/meat.jpg"
        />
        <span
          className="m-4 font-bold text-sm md:text-base "
        >
          Spanish Paella 
        </span>
        <div>
          <img
            alt="chef"
            className="inline p-2 md:w-11 w-8"
            src="images/Avatar.png"
          />
          <span
            className="text-grey-500 md:text-sm font-normal text-xs"
          >
            Jennifer Lawrence
          </span>
        </div>
        <div
          className=" badge"
        >
           
          <span>
            60 tl
          </span>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          alt="meatball"
          className="w-full sm:h-48 object-cover "
          src="images/meat.jpg"
        />
        <span
          className="m-4 font-bold text-sm md:text-base"
        >
          Spanish Paella 
        </span>
        <div>
          <img
            alt="chef"
            className="inline p-2 md:w-11 w-8 "
            src="images/Avatar.png"
          />
          <span
            className="text-grey-500 md:text-sm text-xs"
          >
            Jennifer Lawrence
          </span>
        </div>
        <div
          className="badge"
        >
           
          <span>
            60 tl
          </span>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          alt="meatball"
          className="w-full sm:h-48 object-cover "
          src="images/meat.jpg"
        />
        <span
          className="m-4 font-bold text-sm md:text-base"
        >
          Spanish Paella 
        </span>
        <div>
          <img
            alt="chef"
            className="inline p-2 md:w-11 w-8"
            src="images/Avatar.png"
          />
          <span
            className="text-grey-500 md:text-sm text-xs"
          >
            Jennifer Lawrence
          </span>
        </div>
        <div
          className="badge"
        >
           
          <span>
            60 tl
          </span>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          alt="meatball"
          className="w-full sm:h-48 object-cover "
          src="images/meat.jpg"
        />
        <span
          className="m-4 font-bold text-sm md:text-base"
        >
          Spanish Paella 
        </span>
        <div>
          <img
            alt="chef"
            className="inline p-2 md:w-11 w-8 "
            src="images/Avatar.png"
          />
          <span
            className="text-grey-500 md:text-sm text-xs"
          >
            Jennifer Lawrence
          </span>
        </div>
        <div
          className="badge"
        >
           
          <span>
            60 tl
          </span>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          alt="meatball"
          className="w-full h-32 sm:h-48 object-cover "
          src="images/meat.jpg"
        />
        <span
          className="m-4 font-bold text-sm md:text-base"
        >
          Spanish Paella 
        </span>
        <div>
          <img
            alt="chef"
            className="inline p-2 md:w-11 w-8 "
            src="images/Avatar.png"
          />
          <span
            className="text-grey-500 md:text-sm text-xs"
          >
            Jennifer Lawrence
          </span>
        </div>
        <div
          className="badge"
        >
           
          <span>
            60 tl
          </span>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          alt="meatball"
          className="w-full h-32 sm:h-48 object-cover"
          src="images/meat.jpg"
        />
        <span
          className="m-4 font-bold text-sm md:text-base"
        >
          Spanish Paella 
        </span>
        <div>
          <img
            alt="chef"
            className="inline p-2 md:w-11 w-8 "
            src="images/Avatar.png"
          />
          <span
            className="text-grey-500 md:text-sm text-xs"
          >
            Jennifer Lawrence
          </span>
        </div>
        <div
          className="badge"
        >
           
          <span>
            60 tl
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
